src/main/java/io/pivotal/literx/Part06Request.java

```diff
@@ -17,14 +17,14 @@
 
 //========================================================================================
 
-	// TODO Create a StepVerifier that initially requests all values and expect 4 values to be received
+	// TODO Create a StepVerifier that expects 4 values to be received and requests all remaining values
```

Comment:
this is not a good change. the `initially requests all values` part is important, as a duality to `initially requests 1 value` in the next test.